src/netmessagemaker.h

```diff
@@ -21,6 +21,7 @@ class CNetMsgMaker
         msg.m_type = std::move(msg_type);
         msg.data.reserve(4 * 1024);
```

Comment:
maybe better to drop `reserve` then?

Reply:
don't think so, check commit message [here](https://github.com/dashpay/dash/pull/2672/commits/d812cfc5d112dcad24653c6afc4f3be7c165aecc)